compiler/tests-spec/tests/org/jetbrains/kotlin/spec/checkers/DiagnosticsTestSpecGenerated.java

```diff
@@ -3317,6 +3317,71 @@ public void testAllFilesPresentInOverriding() throws Exception {
             }
         }
 
+        @TestMetadata("compiler/tests-spec/testData/diagnostics/linked/overload-resolution")
```

Comment:
I don't quite understand why this file is changed here

Reply:
My fault. Reverted